lib/livebook/migration.ex

```diff
@@ -172,4 +189,11 @@ defmodule Livebook.Migration do
 
     Map.put(file, :file_system_type, file_system_type)
   end
+
+  defp add_team_hub_offline() do
+    for %{id: "team-" <> _ = id} = attrs <- Storage.all(:hubs),
+        not Map.has_key?(attrs, :offline) do
+      Storage.insert(:hubs, id, offline: false)
+    end
+  end
```

Comment:
@josevalim if you have a preference towards optional attribute let me know :)

Reply:
Optional attribute for sure. I think we need to get used with having partial views of the data.

Reply:
Actually, we need to dump both in case of `true` and `false`, otherwise there's no way to update it (not that we generally would, but I just did that in tests). And if we store both, then I think it's better to deal with normalized data. wdyt?